{
  "uid" : "811519d67e820b33",
  "name" : "test_get_login_submit",
  "fullName" : "test_testcase.TestSingle#test_get_login_submit",
  "historyId" : "9d45b30fd97667aff13f3a162735a3a9",
  "time" : {
    "start" : 1576600899218,
    "stop" : 1576600899582,
    "duration" : 364
  },
  "description" : " 呵呵呵呵1234",
  "descriptionHtml" : "<p>呵呵呵呵1234</p>\n",
  "status" : "failed",
  "statusMessage" : "AssertionError: assert '2' == '0'\n  - 2\n  + 0",
  "statusTrace" : "self = <tests.test_testcase.TestSingle object at 0x10b9c90f0>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            logger.info(ass[\"url\"])\n            logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:74: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_testcase.TestSingle object at 0x10b9c90f0>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            logger.info(ass[\"url\"])\n            logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n                assert ass[\"expected\"] == ass[\"actual\"]\n            except AssertionError:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:76: AssertionError",
  "flaky" : false,
  "beforeStages" : [ {
    "name" : "_Module__pytest_setup_module",
    "time" : {
      "start" : 1576600898184,
      "stop" : 1576600898184,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1576600898184,
      "stop" : 1576600898184,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : " 呵呵呵呵1234",
    "status" : "failed",
    "statusMessage" : "AssertionError: assert '2' == '0'\n  - 2\n  + 0",
    "statusTrace" : "self = <tests.test_testcase.TestSingle object at 0x10b9c90f0>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            logger.info(ass[\"url\"])\n            logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:74: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_testcase.TestSingle object at 0x10b9c90f0>\n\n    @allure.story(\"测试3\")\n    def test_get_login_submit(self):\n        \"\"\" 呵呵呵呵1234\"\"\"\n        single_testcase_yaml = os.path.join(os.path.dirname(__file__), \"api\", \"get_login_submit.yml\")\n    \n        result = self.run.run_yml(single_testcase_yaml)\n        logger.info(\"666666666\"+str(result))\n        for ass in result[0]:\n            logger.info(ass[\"url\"])\n            logger.info(\"\\033[1;31m=======Assert=====\\033[0m\")\n            logger.info(\"\\033[1;36mexpected:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"expected\", ass[\"expected\"]))\n            logger.info(\"\\033[1;32mactual:\\033[0m\" + '\\n\\t' + \"{}={}\".format(\"actual\", ass[\"actual\"]))\n            try:\n                assert ass[\"expected\"] == ass[\"actual\"]\n            except AssertionError:\n>               assert ass[\"expected\"] == ass[\"actual\"]\nE               AssertionError: assert '2' == '0'\nE                 - 2\nE                 + 0\n\ntest_testcase.py:76: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "806349615e1ca035",
      "name" : "log",
      "source" : "806349615e1ca035.txt",
      "type" : "text/plain",
      "size" : 2205
    }, {
      "uid" : "6fc9b94b922c9b85",
      "name" : "stdout",
      "source" : "6fc9b94b922c9b85.txt",
      "type" : "text/plain",
      "size" : 0
    }, {
      "uid" : "d4664584e9f123f7",
      "name" : "stderr",
      "source" : "d4664584e9f123f7.txt",
      "type" : "text/plain",
      "size" : 0
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "_Module__pytest_setup_module::0",
    "time" : {
      "start" : 1576600900014,
      "stop" : 1576600900014,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_Class__pytest_setup_class::0",
    "time" : {
      "start" : 1576600900013,
      "stop" : 1576600900013,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "story",
    "value" : "测试3"
  }, {
    "name" : "feature",
    "value" : "测试接口"
  }, {
    "name" : "suite",
    "value" : "test_testcase"
  }, {
    "name" : "subSuite",
    "value" : "TestSingle"
  }, {
    "name" : "host",
    "value" : "anxiaodongdeMacBook-Pro.local"
  }, {
    "name" : "thread",
    "value" : "5992-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_testcase"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "811519d67e820b33.json",
  "parameterValues" : [ ]
}